File: TetJava/Blocks/TBlock.java
package TetJava.Blocks;

/*

Tblock can be printed in 4 ways:

[]
[][]
[]

[][][]
  []

  []
[][][]

[]
[][]
[]

*/
public class TBlock extends BlockABC{

    // constructor
    public TBlock() {

        this.orientations = new int[][][] {
            /*
            []
            [][]
            []
            */
            {
                {0, 0},
                {0, 1},
                {1, 1},
                {0, 2}
            },
            /*
              []
            [][]
              []
             */
            {
                {1, 0},
                {1, 2},
                {1, 1},
                {0, 1}
            },
            /*
              []
            [][][]
             */
            {
                {1, 0},
                {0, 1},
                {1, 1},
                {2, 1}
            },
            /*
            [][][]
              []
             */
            {
                {0, 0},
                {1, 0},
                {2, 0},
                {1, 1}
            }
        };

        // places the default T shape
        this.partCoords = orientations[0];

        // sets the starting point for the block
        this.xpos = 4;
        this.ypos = -1;
    }
}

File: TetPy/Blocks/tblock.py
from Blocks.block_abc import Block_ABC

class Tblock(Block_ABC):
    
    def __init__(self, board_width, board_height):
        super(Tblock, self).__init__(board_width, board_height)
        self.xpos = 3
        self.ypos = 0
        self.orientations = [
            # [][][]
            #   []
            [
                [0, 0],
                [1, 0],
                [2, 0],
                [1, 1]
            ],
            #   []
            # [][]
            #   []
            [
                [0, 1],
                [1, 0],
                [1, 1],
                [1, 2]
            ],
            #   []
            # [][][]
            [
                [1, 0],
                [0, 1],
                [1, 1],
                [2, 1]
            ],
            # []
            # [][]
            # []
            [
                [0, 0],
                [0, 1],
                [0, 2],
                [1, 1]

            ]
        ]
        self.part_coords = self.orientations[0]